src/index_registry.cpp

```diff
@@ -319,7 +314,7 @@ vector<string> vcf_contigs(const string& filename) {
     
     htsFile* vcf = hts_open(filename.c_str(),"rb");
     if (vcf == nullptr) {
-        cerr << "error:[IndexRegistry] Could not open VCF" << filename << endl;
+        error(context) << "Could not open VCF: " << filename << endl;
```

Comment:
I guess having `error()` exit helps us avoid cases like we're fixing here, where an error is detected but not acted on.

But now that this is so simple and in some ways so much like Python logging it's a little astonishing that that happens. Maybe we just have to see if it actually trips people up ever?